<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>847</width>
    <height>695</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Spreadsheet</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Spreadsheet.qrc">
    <normaloff>:/images/Exit.bmp</normaloff>:/images/Exit.bmp</iconset>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>847</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <widget class="QMenu" name="menu_Select">
     <property name="title">
      <string>&amp;Select</string>
     </property>
     <addaction name="actionColumn"/>
     <addaction name="action_Row"/>
     <addaction name="action_All"/>
    </widget>
    <addaction name="action_Cut"/>
    <addaction name="action_Copy"/>
    <addaction name="action_Paste"/>
    <addaction name="action_Delete"/>
    <addaction name="menu_Select"/>
    <addaction name="separator"/>
    <addaction name="action_Find"/>
    <addaction name="action_Go_to_Cell"/>
   </widget>
   <widget class="QMenu" name="menu_Tools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="action_Recalculate"/>
    <addaction name="action_Sort"/>
   </widget>
   <widget class="QMenu" name="menu_Options">
    <property name="title">
     <string>&amp;Options</string>
    </property>
    <addaction name="action_Show_Grid"/>
    <addaction name="action_Auto_Recalculate"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_Tools"/>
   <addaction name="menu_Options"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="fileToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="editToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Cut"/>
   <addaction name="action_Copy"/>
   <addaction name="action_Paste"/>
   <addaction name="separator"/>
   <addaction name="action_Find"/>
   <addaction name="action_Go_to_Cell"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="Spreadsheet.qrc">
     <normaloff>:/images/New.bmp</normaloff>:/images/New.bmp</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="toolTip">
    <string>Create</string>
   </property>
   <property name="statusTip">
    <string>Create a new spreadsheet file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="Spreadsheet.qrc">
     <normaloff>:/images/open.gif</normaloff>:/images/open.gif</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="toolTip">
    <string>Open</string>
   </property>
   <property name="statusTip">
    <string>Open a exiting spreadsheet file</string>
   </property>
   <property name="shortcut">
    <string>Alt+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="Spreadsheet.qrc">
     <normaloff>:/images/Save.bmp</normaloff>:/images/Save.bmp</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="statusTip">
    <string>Save the spreadsheet file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save &amp;As</string>
   </property>
   <property name="statusTip">
    <string>Save the spreadsheet under a new name</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="statusTip">
    <string>Exit the application</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_Cut">
   <property name="icon">
    <iconset resource="Spreadsheet.qrc">
     <normaloff>:/images/Cut.bmp</normaloff>:/images/Cut.bmp</iconset>
   </property>
   <property name="text">
    <string>&amp;Cut</string>
   </property>
   <property name="statusTip">
    <string>Cut the current selected to a new board</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="action_Copy">
   <property name="icon">
    <iconset resource="Spreadsheet.qrc">
     <normaloff>:/images/Copy.bmp</normaloff>:/images/Copy.bmp</iconset>
   </property>
   <property name="text">
    <string>&amp;Copy</string>
   </property>
   <property name="statusTip">
    <string>Copy current selection's content to the board</string>
   </property>
  </action>
  <action name="action_Paste">
   <property name="icon">
    <iconset resource="Spreadsheet.qrc">
     <normaloff>:/images/Paste.bmp</normaloff>:/images/Paste.bmp</iconset>
   </property>
   <property name="text">
    <string>&amp;Paste</string>
   </property>
   <property name="statusTip">
    <string>Paste the clipboard's contents into the current ...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="action_Delete">
   <property name="text">
    <string>&amp;Delete</string>
   </property>
   <property name="statusTip">
    <string>Delete the current selections</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="action_Find">
   <property name="icon">
    <iconset resource="Spreadsheet.qrc">
     <normaloff>:/images/find.gif</normaloff>:/images/find.gif</iconset>
   </property>
   <property name="text">
    <string>&amp;Find</string>
   </property>
   <property name="statusTip">
    <string>Find a matching cell</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="action_Recalculate">
   <property name="text">
    <string>&amp;Recalculate</string>
   </property>
   <property name="statusTip">
    <string>Recalculate all the shreadsheet's value</string>
   </property>
   <property name="shortcut">
    <string>F12</string>
   </property>
  </action>
  <action name="action_Sort">
   <property name="text">
    <string>&amp;Sort</string>
   </property>
   <property name="statusTip">
    <string>Sort the select cells or all the cells</string>
   </property>
  </action>
  <action name="action_Show_Grid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Show Grid</string>
   </property>
   <property name="statusTip">
    <string>Show or hide the spreadsheet</string>
   </property>
  </action>
  <action name="action_Auto_Recalculate">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Auto-Recalculate</string>
   </property>
   <property name="statusTip">
    <string>Switch auto-recalculate on the spreadsheet</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About</string>
   </property>
   <property name="statusTip">
    <string>Show the applicatin tisps</string>
   </property>
  </action>
  <action name="actionColumn">
   <property name="text">
    <string>&amp;Column</string>
   </property>
   <property name="statusTip">
    <string>Select all the cells current column</string>
   </property>
  </action>
  <action name="action_Row">
   <property name="text">
    <string>&amp;Row</string>
   </property>
   <property name="statusTip">
    <string>Select all the cells current row</string>
   </property>
  </action>
  <action name="action_All">
   <property name="text">
    <string>&amp;All</string>
   </property>
   <property name="statusTip">
    <string>Select all the cells in the spreadsheet </string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="action_Go_to_Cell">
   <property name="icon">
    <iconset resource="Spreadsheet.qrc">
     <normaloff>:/images/gotocell.gif</normaloff>:/images/gotocell.gif</iconset>
   </property>
   <property name="text">
    <string>&amp;Go to Cell</string>
   </property>
   <property name="statusTip">
    <string>Go to the specified cell</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Spreadsheet.qrc"/>
 </resources>
 <connections/>
</ui>
